libnmstate/nm/team.py

```diff
@@ -88,27 +88,32 @@ def get_info(device):
         teamd_json = device.get_config()
         if teamd_json:
             teamd_config = json.loads(teamd_json)
-            team_config = _convert_teamd_config_to_nmstate_config(teamd_config)
+            team_config = _convert_teamd_config_to_nmstate_config(
+                teamd_config, device
```

Comment:
Passing the list of slaves names will be nicer IMO. Letting this functions know about the NM device adds a burden which is not needed.

Reply:
I agree, thanks!